src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -86,7 +86,7 @@ class HaConfigDashboard extends LitElement {
                     </ha-card>`
                   : ""}
                 <ha-card>
-                  ${this.narrow && this.supervisorUpdates !== null
+                  ${this.narrow && this.supervisorUpdates?.length
```

Comment:
Shouldn't we put this guard on line 79 ? We don't want to render any card right ?

Reply:
Yes, but it needs to be here as well, this is for the "Configuration" header that show on mobile